### PR TITLE
[Routing] Document the new alias feature

### LINKED ISSUE #16070: [Routing] Add support for aliasing routes
| Q            | A
| ------------ | ---
| Feature PR   | symfony/symfony#38464
| PR author(s) | @Lctrs
| Merged in    | 5.4

### PR DESCRIPTION
**Pull request by Lctrs**
Closes #16070.

### CODE REVIEW

File: routing.rst [hunk 1]
```diff
@@ -1453,6 +1453,117 @@ A possible solution is to change the parameter requirements to be more permissiv
...
     the ``.+`` requirement by ``[^.]+`` to allow any character except dots.
 
+.. _routing-alias:
+
+Aliasing
+--------
+
+.. versionadded:: 5.4
+
+    Aliases support was introduced in Symfony 5.4.
```
Comment (OskarStark) [inline routing.rst]:
```suggestion
    Support for route aliases was introduced in Symfony 5.4.
```

File: routing.rst [hunk 2]
```diff
@@ -1453,6 +1453,117 @@ A possible solution is to change the parameter requirements to be more permissiv
...
+.. _routing-alias:
+
+Aliasing
+--------
+
+.. versionadded:: 5.4
+
+    Support for route aliases was introduced in Symfony 5.4.
+
+You may sometimes want to have multiple ids for the same route. You can do so by
```
Comment (94noni) [inline routing.rst]: IMO the word « ids » is not the best
On routing doc it is named « route name »
Just my 2cts :)

Comment (Lctrs) [reply routing.rst]: Updated.

**Comment by Lctrs** [comment]
@OskarStark Do you have any more review to make ?

**Review by 94noni — APPROVED** [review]

**Comment by OskarStark** [comment]
No this looks good to me 👍🏻

**Review by OskarStark — APPROVED** [review]

**Comment by javiereguiluz** [comment]
@Lctrs we're sorry it took us so long to merge this ... and thanks a lot for your contribution!

**Pull request merged**